ivy-scaling-nginx/docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: '3.7'
+services:
+  loadbalancer:
+    image: nginx:stable-alpine
+    ports:
+      - 80:80
+    volumes:
+      - ./nginx.conf:/etc/nginx/nginx.conf
+    depends_on:
+      # Ensures that ivy cluster is started before nginx
+      - ivy
+
+  ivy:
+    image: axonivy/axonivy-engine:dev
+    ports:
+      - 8080:8080   # Note: This needs to be removed if there is more than one node configured

```

Comment:
there should be no need to mount the port to the outside world?!

Reply:
The 8080 port needs to be visible for our tests.
But I found a better way by configuring the loadbalancer with ports: - 8080:80

Reply:
Can't the test work with port 80?

Reply:
Port 8080 is always used to check if the Engine is up-and-running.
The assertion / test itself works with port 80 (see Jenkinsfile).

Reply:
Ok, not needed, I removed it.